src/docker/Dockerfile

```diff
@@ -26,6 +26,9 @@ COPY python/requirements-app.txt ./
 RUN --mount=type=cache,target=/root/.cache  \
     { source activate rapids || echo ok ; } && pip install -r requirements-app.txt
 
+RUN --mount=type=cache,target=/root/.cache  \
+    { source activate rapids || echo ok ; } && pip install --no-deps streamlit pympler altair watchdog
```

Comment:
pympler, altair and watchdog were dependencies of streamlit that were needed from adding the `--no-deps` option